src/routes/chat/SocketService/socketSetup.js

```diff
@@ -0,0 +1,15 @@
+import io from 'socket.io-client';
+
+const Socket = {
+  socket: null,
+  connect: null,
+  load: null
+};
+
+const connect = () => {
+  Socket.socket = io.connect(`${process.env.REACT_APP_BACKEND_API}chat`);
+};
+
+connect();
```

Comment:
don't call it here directly, this means once the app starts up the socket will connect. Only call it onEffect in Chat.jsx or something.